### DISCUSSION TITLE
No indication of a text selection

### DISCUSSION THREAD

**Discussion by BMJ-pdx** [kovidgoyal/kitty#3797] [Q&A] [answered]
In kitty 0.21.2 running under Fedora, with a default kitty.conf, there is no indication of what text has been selected with the mouse.  It is really selected; i.e. it can be pasted.
I can change selection foreground/background colors as shown in FAQ "How do I change the colors in a running kitty instance?", and then a selection is highlighted as expected.
What's wrong here?  Surely there should be some indication by default.

**Comment by kovidgoyal** [comment, marked as answer]
On Sun, Jul 04, 2021 at 08:12:04PM -0700, BMJ-pdx wrote:
> In kitty 0.21.2 running under Fedora, with a default kitty.conf, there is no indication of what text has been selected with the mouse.  It is really selected; i.e. it can be pasted.
> I can change selection foreground/background colors as shown in FAQ "How do I change the colors in a running kitty instance?", and then a selection is highlighted as expected.
> What's wrong here?  Surely there should be some indication by default.

Presumably you have changed your kitty background color to be the same
as the highlight background color in kitty.conf. If you are going to do
that, change the highlight background color to something else as well.

**Reply by BMJ-pdx** [reply]
Yes, I have changed to black text on a white background.  I apparently missed the conflict with the highlight colors; I was searching for something specific to selection.
Thanks for the speedy (!) response.
PS.:  And an apology for saying I had a stock kitty.conf -- that was not the case.
PPS.:  Good grief.  I now see "selection_foreground" and "selection_background" in kitty.conf .  I don't know how I missed those.  So, I'm sorry for the stupid query.
